format/format.go

```diff
@@ -1,26 +0,0 @@
-// Package format defines any necessary custom formatters for the responses
-// presently only JSON is supported.
-package format
-
-import (
-	"encoding/json"
-	"net/http"
-
-	"github.com/LUSHDigital/microservice-core-golang/response"
-)
-
-// JSONResponseFormatter formats a microservice response as JSON.
-func JSONResponseFormatter(w http.ResponseWriter, response response.ResponseInterface) error {
```

Comment:
This is the first time I'm seeing some of this, so it's probably a dumb question but do we need this?  There's no equivalent in the new code.

Reply:
The equivalent is effectively the `WriteTo` in `response.go` @codingconcepts

Reply:
Precicely, since we were only using that one method, I don't see a point in keeping the format package at all